src/ontology/edits/oeo-shared.omn

```diff
@@ -1119,6 +1175,9 @@ Class: OEO_00000064
         OEO_00000051
     
     
+Class: OEO_00000087
```

Comment:
![grafik](https://github.com/OpenEnergyPlatform/ontology/assets/36884905/47a09a5e-10bc-4ca8-9997-931b7a0b32d1)

```suggestion
```
This class declaration is not needed anymore.